tests/manage/test_ocs_288.py

```diff
@@ -50,17 +50,22 @@ def teardown():
     """
     global RBD_SECRET_OBJ, CEPHFS_SECRET_OBJ
     log.info("Deleting PVC")
-    assert pvc.delete_all_pvcs()
+    for count in range(len(PVC_OBJ)):
```

Comment:
what are you trying to achieve here?
I guess you want to do:
```python
for pvc im PVC_OBJ:
    assert pvc.delete_pvcs(pvc.name)
```

In case that's true, and PVC_OBJ is a list of pvc object, please change the name as it is very confusing

Reply:
Sorry my bad fix in latest commit